geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/ScaldingDelimitedIngestJob.scala

```diff
@@ -142,7 +143,7 @@ class ScaldingDelimitedIngestJob(args: Args) extends Job(args) with Logging {
     }
   }
 
-  def ingestLine(fw: FeatureWriter[SimpleFeatureType, SimpleFeature], line: String): Unit =
+  def ingestLine(fw: FeatureWriter[SimpleFeatureType, SimpleFeature], line: String, skipHeader: Boolean = false): Unit =
     Try {
       ingestDataToFeature(line, fw.next())
```

Comment:
I'm thinking we should never call ```ingestDataToFeature()``` if we are skipping the header...it's an anti-pattern to use exceptions as control flow.